youtube_transcript_api/_api.py

```diff
@@ -104,7 +118,23 @@ def get_transcript(cls, video_id, languages=('en',), proxies=None):
         :type languages: list[str]
         :param proxies: a dictionary mapping of http and https proxies to be used for the network requests
         :type proxies: {'http': str, 'https': str} - http://docs.python-requests.org/en/master/user/advanced/#proxies
+        :param cookies: a string of the path to a text file containing youtube authorization cookies
+        :type cookies: str
         :return: a list of dictionaries containing the 'text', 'start' and 'duration' keys
         :rtype [{'text': str, 'start': float, 'end': float}]:
         """
-        return cls.list_transcripts(video_id, proxies).find_transcript(languages).fetch()
+        return cls.list_transcripts(video_id, proxies, cookies).find_transcript(languages).fetch()
+    
+    @classmethod
+    def load_cookies(cls, cookies):
```

Comment:
I don't think that `load_cookies` should be part of the public API, as this doesn't provide relevant functionality to the user. So maybe make this private or keep it inlined.